tests/utils/test_hf_argparser.py

```diff
@@ -379,7 +379,7 @@ def test_parse_json(self):
             os.mkdir(temp_local_path)
             with open(temp_local_path + ".json", "w+") as f:
                 json.dump(args_dict_for_json, f)
-            parsed_args = parser.parse_yaml_file(Path(temp_local_path + ".json"))[0]
+            parsed_args = parser.parse_json_file(Path(temp_local_path + ".json"))[0]
```

Comment:
This shouldn't be changed

Reply:
I have reverted this commit

Reply:
Sorry, brain blup on my end - your completely right - the test should use `parse_json_file`